src/acceptance/app/app_suite_test.go

```diff
@@ -208,3 +240,102 @@ func generateDynamicScaleInPolicy(instanceMin, instanceMax int, threshold int64)
 
 	return string(bytes)
 }
+
+func generateDynamicAndSpecificDateSchedulePolicy(instanceMin, instanceMax int, threshold int64,
```

Comment:
The policy has both dynamic scaling and scheduled scaling rules.. It is a test of scheduled scaling with dynamic scaling during the scheduled time period.